tools/busywork/tcl/fabric.tcl

```diff
@@ -357,7 +357,7 @@ proc ::fabric::caLogin {i_peer i_user i_secret} {
     http::cleanup $token
 }
 
-    
```

Comment:
Prelude does whitespace-cleanup by default 
http://batsov.com/prelude/ 
Look for "whitespace-mode", tells how to disable it if you want to